redash/monitor.py

```diff
@@ -26,7 +26,7 @@ def get_object_counts():
     return status
 
 
-def get_queues():
+def get_celery_queues():
```

Comment:
This function is also used in `celery_tasks()`, so need to update the name there as well.

Reply:
D'oh!